docs/tasks.py

```diff
@@ -0,0 +1,27 @@
+"""Tasks used with invoke for docs."""
+from invoke import task
+
+
+@task
+def html(context, sourcedir="source", builddir="build"):
+    """Creates html docs using sphinx-build command.
+
+    Args:
+        context (obj): Used to run specific commands
+        sourcedir (str, optional): Source directory for sphinx to use. Defaults to "source".
+        builddir (str, optional): Output directory for sphinx to use. Defaults to "build".
+    """
+    command = "sphinx-build" + sourcedir + builddir
```

Comment:
I guess I am missing something, but to my untrained eye it seems like this is missing whitespace.

Reply:
You are correct. Updated.